src/Components/PriceHistory/PriceHistory.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Col, Table } from "react-bootstrap";
+import './PriceHistory.css';
+
+function PriceHistory() {
+
+    return (
+        <div className="PriceHistoryComponent">
```

Comment:
This is a static component, I recommend figuring out how to pass in the array of data required to show the component and making it dynamic via props.

https://stackoverflow.com/questions/43948828/how-to-pass-an-array-of-items-in-react-js